shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/kernel/model/ExecutionGroupContext.java

```diff
@@ -38,7 +38,7 @@
     
     private final Collection<ExecutionGroup<T>> inputGroups;
     
-    private final String executionID = new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString();
+    private final String executionID = new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString().replaceAll("-", "");
```

Comment:
Why add `.replaceAll("-", "")` for `executionID`?

Reply:
DALStatement.g4 use `KILL (CONNECTION | QUERY)? IDENTIFIER_`, `IDENTIFIER_` not contains `-`